src/providers/commandPalette.ts

```diff
@@ -41,8 +95,7 @@ const useCommandPaletteBase = () => {
 
   const openCommandPalette = (mode: CommandPaletteMode, initialInput = '') => {
     commandPaletteStore.mode = mode
-    commandPaletteStore.query = ''
-    commandPaletteStore.currentInput = initialInput
+    if (initialInput !== '') commandPaletteStore.currentInput = initialInput
```

Comment:
副作用起こす場合はブロックにしてほしいです :pray:
```suggestion
    if (initialInput !== '') {
      commandPaletteStore.currentInput = initialInput
    }
```